### PR TITLE
Fix image coding in Cloud SIEM administration articles

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) fixes image coding and does other cleanup work in the articles in this section:
https://help.sumologic.com/docs/cse/administration/

Issue number: Asana ticket "[Clean up links and image coding in SIEM and SOAR](https://app.asana.com/0/1183741177809740/1204726032012276/f)"

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/administration/create-a-custom-tag-schema.md [hunk 1]
```diff
@@ -44,6 +36,4 @@ For more information about tags in CSE, see [Using Tags with Insights, Signals,
     1. **Value Options**. If **Allow Custom Values** is not checked, you must define at least one value for the tag:
         * **Value**. Enter an allowable value for the tag.
         * **Label**. Enter a label for the value.
-        * **Link**. If desired, enter a URL. If you supply a URL, that link will appear in the Actions menu of the tag in any content items to which it’s been applied.  CSE’s built-in schema tags are examples of schema tags that include a link. The screenshot below shows a link from the **Tactic:TA0002** to associated information on the MITRE site.
-
-            ![mitre-link.png](/img/cse/mitre-link.png)
+        * **Link**. If desired, enter a URL. If you supply a URL, that link will appear in the Actions menu of the tag in any content items to which it’s been applied.  CSE’s built-in schema tags are examples of schema tags that include a link. The screenshot below shows a link from the **Tactic:TA0002** to associated information on the MITRE site. <br/><img src={useBaseUrl('img/cse/mitre-link.png')} alt="MITRE link" width="800"/>
```
Comment (kimsauce) [inline docs/cse/administration/create-a-custom-tag-schema.md:39]:
```suggestion
        * **Link** (optional). Enter a URL for it to appear in the Actions menu of the tag in any content items to which it’s been applied. CSE’s built-in schema tags are examples of schema tags that include a link. The screenshot below shows a link from the **Tactic:TA0002** to associated information on the MITRE site. <br/><img src={useBaseUrl('img/cse/mitre-link.png')} alt="MITRE link" width="800"/>
```

File: docs/cse/administration/create-custom-threat-intel-source.md [hunk 2]
```diff
@@ -39,23 +42,18 @@ Rule authors can also write rules that look for threat intelligence information
...
-1. Click **Add Source** on the **Threat Intelligence** page.  
-    ![threat-intel-page2.png](/img/cse/threat-intel-page2.png)
-1. Click **Custom** on the **Add Source** popup.  
-    ![custom-button.png](/img/cse/custom-button.png)
+1. Click **Add Source** on the **Threat Intelligence** page. <br/><img src={useBaseUrl('img/cse/threat-intel-page2.png')} alt="Threat Intelligence page" width="800"/>
+1. Click **Custom** on the **Add Source** popup. <br/><img src={useBaseUrl('img/cse/custom-button.png')} alt="Custom button" width="600"/>
 1. On the **Add New Source** popup, enter a name, and if desired, a
-    description for the source.  
-    ![add-custom-source.png](/img/cse/add-custom-source.png)
+    description for the source. <br/><img src={useBaseUrl('img/cse/add-custom-source.png')} alt="Add new source" width="600"/>
```
Comment (kimsauce) [inline docs/cse/administration/create-custom-threat-intel-source.md:48]: this needs to be joined with the previous line

**Review by kimsauce — CHANGES_REQUESTED** [review]
a couple of suggestions

File: docs/cse/administration/create-custom-threat-intel-source.md (hunk 2 above)
Comment (jpipkin1) [reply docs/cse/administration/create-custom-threat-intel-source.md]: Fixed.

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**